quarkus/config-api/src/main/java/org/keycloak/config/CachingOptions.java

```diff
@@ -62,6 +65,7 @@ public String toString() {
     public static final Option<Stack> CACHE_STACK = new OptionBuilder<>("cache-stack", Stack.class)
             .category(OptionCategory.CACHE)
             .expectedValues(false)
+            .deprecatedValues(Stream.of(Stack.azure, Stack.ec2, Stack.google).map(Object::toString).collect(Collectors.toSet()), "Use 'jgroups-ping' instead")
```

Comment:
`jgroups-ping' should be 'jdbc-ping'. See also my fix in the depending PR. Consider using the cache stack type enum to prepare the text.